src/include/target/NamimnoRC_VOYAGER_900_OLED_TX.h

```diff
@@ -43,4 +43,7 @@
 #define POWER_OUTPUT_DACWRITE
 #define MinPower                PWR_25mW
 #define MaxPower                PWR_2000mW
-#define POWER_OUTPUT_VALUES     {57,67,76,90,112,132,167,201}
\ No newline at end of file
+#define POWER_OUTPUT_VALUES     {57,67,76,90,112,132,167,201}
+
+/* Joystick values              {UP, DOWN, LEFT, RIGHT, ENTER, IDLE}*/
+#define JOY_ADC_VALUES          {1850, 900, 490, 1427, 0, 2978}
```

Comment:
@FOG-Yamato same here just like the 2400 target. 

Are these the right ADC Values??

Reply:
Same as the 2400 target.